src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,12 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine[] machines = new Machine[]{new Truck(), new Bulldozer(), new Excavator()};
```

Comment:
you can omit `new Machine[]`

but it's the matter of taste, don't consider it as critical :)
```suggestion
        Machine[] machines = {new Truck(), new Bulldozer(), new Excavator()};
```